ui/app/models/pki/certificate/base.js

```diff
@@ -48,9 +48,9 @@ export default class PkiCertificateBaseModel extends Model {
   @attr('string') serialNumber;
 
   // Parsed from cert in serializer
-  @attr('number', { formatDate: true }) issueDate;
-  @attr('number', { formatDate: true }) notValidAfter;
-  @attr('number', { formatDate: true }) notValidBefore;
+  @attr('date', { formatDate: true }) issueDate;
+  @attr('date', { formatDate: true }) notValidAfter;
+  @attr('date', { formatDate: true }) notValidBefore;
```

Comment:
I changed these to `number` when I noticed that they were coming back from the parser as numbers. Date more accurately describes the attr type and I'm not seeing any issues in the certificate read view with the date transform applied. The `formatDate` option can be removed from all 3 of these and on line 35 of `pki-certificate-details.hbs` the condition could be changed to look for `(eq field.type "date")` instead.

Reply:
Ah, I changed this because the issuer details view wasn't being formatted correctly

Reply:
Thanks for the context! I'm confused by your suggestion 🤔 If I do all of that then none of the dates are formatted in the details view for issuers `notValidAfter` and all the certificates dates

Reply:
It looks like issuer and certificates are just handling the same attr differently - pushing up a change! let me know what you think :)

Reply:
I tested locally in `pki-certificate-details` and the result is the same. It is trivial so we can leave it the way it is but `formatDate` as an attr option was only added because the attr `type` was number. Now that they are all dates it could be removed and the attr type could be used to check if it is date.

Reply:
It looks like you reverted the type to number so my previous comment is no longer relevant. I was suggesting that if we were to change to date type then there is no need to have the `formatDate` attr option and could simply look for the attr type date to determine whether or not it should be formatted as a date in the template.